README.md

```diff
@@ -11,6 +11,7 @@ OpenEBS is a modern Block-Mode storage platform, a Hyper-Converged software Stor
 ## Monthly Community Meetings
 The community meetings are usually held monthly on the last thursday of the month at 14:00 UTC. <BR>
 The meeting is currently held on google meet: https://meet.google.com/oaa-fycu-vsq <BR>
+Next meeting is scheduled for **8th Aug 2024 at 14:00 UTC**. <BR>
```

Comment:
That is on the invite, but I put the wrong link above, should be:
https://calendar.app.google/TBQthr3oW42uyJqz5

Reply:
Resolved with the latest commit.